spring-boot-project/spring-boot-docker-compose/src/dockerTest/resources/org/springframework/boot/docker/compose/service/connection/db2/db2-compose.yaml

```diff
@@ -0,0 +1,20 @@
+services:
+  database:
+    image: '{imageName}'
+    ports:
+      - '50000:50000'
+    privileged: true
+    environment:
+      - 'LICENSE=accept'
+      - 'DB2INSTANCE=db2inst1'
+      - 'DB2INST1_PASSWORD=secret'
+      - 'DBNAME=testdb'
+      - 'AUTOCONFIG=false'
+      - 'ARCHIVE_LOGS=false'
+    healthcheck:
+      test: [ 'CMD', 'sleep', '115' ]
```

Comment:
Didn't find better way for waiting DB2 fully up.

Reply:
Testcontainers `Db2Container` use `LogMessageWaitStrategy() .withRegEx(".*Setup has completed\\..*")` , is there an equivalent way with compose file?